doc/stdlib/index-list.html.template

```diff
@@ -531,6 +531,7 @@ through the <tt>Require Import</tt> command.</p>
     theories/Reals/Cauchy/ConstructiveCauchyReals.v
     theories/Reals/Cauchy/ConstructiveCauchyRealsMult.v
     theories/Reals/Cauchy/ConstructiveCauchyAbs.v
+    theories/Reals/Cauchy/ConstructiveCauchyRealsPerformance.v
```

Comment:
I want to keep those Cauchy reals to a bare minimum. They are merely a proof that structure `ConstructiveReals` is inhabited. They should not be imported either, because we recommend to use universal quantifications over `ConstructiveReals` instead.

If this file was really intended to provide square root, it would need a proof that its square is equal to the initial rational number. It would also need an extension to all real numbers. And all those should be done in CoRN rather than stdlib. This file is only intended for speed meaures.